exchanges/binance/cfutures_types.go

```diff
@@ -204,21 +205,39 @@ type BatchCancelOrderData struct {
 	Msg           string  `json:"msg"`
 }
 
+type futuresNewOrderRequest struct {
```

Comment:
For library use, this struct and its properties should be exported. Please also add a comment to the struct